external/fv3fit/fv3fit/keras/_models/loss.py

```diff
@@ -1,148 +0,0 @@
-from typing import Callable, Union, Mapping, MutableMapping
```

Comment:
It's nice to delete this, it was kind of confusing when both this and `_models/shared/loss.py` were around.